Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -0,0 +1,145 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import UIKit
+
+internal class RUMViewScope: RUMScope {
+    // MARK: - Initialization
+
+    private unowned let parent: RUMScope
+    private let dependencies: RUMScopeDependencies
+
+    /// Weak reference to the `UIViewController` which issued this scope.
+    private(set) weak var identity: AnyObject?
+    /// View attributes.
+    private(set) var attributes: [AttributeKey: AttributeValue]
+
+    /// This View UUID.
+    let viewUUID: RUMUUID
+    /// The URI of this View, used as the `view.url` in RUM Explorer.
+    let viewURI: String
+    /// The start time of this View.
+    private var viewStartTime: Date
+
+    /// Number of actions happened on this View.
+    private var actionsCount: UInt = 0
+    /// Current version of this View to use for RUM `documentVersion`.
+    private var version: UInt = 0
```

Comment:
this is not clear to me 🤔 what does version of view mean?

Reply:
Whenever something happens on the View (i.e. Resource finishes loading or User Action is observed), a new `RUMViewEvent` is send to the backend. The `event.dd.documentVersion` is incremented for every new event of the View. This way, backend can reduce received events (View updates) into single RUM View displayed in RUM Explorer.

This `version` property tracks the `documentVersion` of this View.